src/App.js

```diff
@@ -32,11 +37,39 @@ function App() {
     fetchMovieData();
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+  useEffect(() => {
+    setError(null);
+    if (search === '') {
+      return;
+    }
+
+    async function fetchSearchMovieData() {
+      const API_KEY = process.env.REACT_APP_API_KEY;
```

Comment:
You don't need this here as you already got your API key in line 11
```suggestion
```

Reply:
I thought since const is block-scoped I have to declare the key separate?

Reply:
No :)

Reply:
The `API_KEY` variable is declared in your `App` function, so within this function, the variable is _always_ reachable :)